src/components/Tabs/Annotation/AnnotationDisplay/ListItems/PlaceholderListItem.tsx

```diff
@@ -9,9 +9,9 @@ export default function PlaceholderListItem(): ReactElement {
   const theme = useContext(AppThemeContext);
   return (
     <FlexRowAlignCenter style={{ width: "100% ", height: "100px" }}>
-      <FlexColumnAlignCenter style={{ margin: "16px 0 10px 0", width: "100%", color: theme.color.text.disabled }}>
+      <FlexColumnAlignCenter style={{ margin: "16px 20px 10px 0", width: "100%", color: theme.color.text.disabled }}>
```

Comment:
Horizontally centers the placeholder seen when there are no annotations:

![image](https://github.com/user-attachments/assets/8beb1efb-5547-4d90-84ae-f0273ba6828a)